### PR TITLE
Simplify installation instructions

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

## Description

The project evolved over time and Thamos CLI provides sub-commands that can be used to install and run the application. Respect this in the README instructions.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -32,57 +32,40 @@ The pre-configured template for Thamos CLI is available in the
...
-be used for running the application, create a Python environment and install
-requirements into it:
+be used for running the application, you can use Thamos to create a Python
+environment (based on configuration in ``.thoth.yaml``) and install the
+recommended requirements into it:
 
 .. code-block:: console
 
-  pip3 install micropipenv
+  thamos install
```
Comment (mayaCostantini) [inline README.rst:48]: Is `micropipenv` already in the requirements?

**Review by mayaCostantini — COMMENTED** [review]
lgtm :+1:

File: README.rst (hunk 1 above)
Comment (fridex) [reply README.rst:48]: Yes, it is a dependency of thamos:

https://github.com/thoth-station/thamos/blob/2fe398dcfbeb8477305aa285be550e565e9a51a8/requirements.txt#L18

But the user does not need to use it at all when using thamos (thamos wraps it with a few checks to properly integrate it).

**Pull request merged**